src/Service/Coins/CoinsCreateService.php

```diff
@@ -23,37 +25,43 @@ class CoinsCreateService
     private $em;
 
     /**
-     * @param CoinsParseService $coinsParseService
+     * @param CoinsGeckoClient $coinsGeckoClient
      * @param CoinRepository $coinRepository
      * @param EntityManagerInterface $em
      */
     public function __construct(
-        CoinsParseService $coinsParseService,
+        CoinsGeckoClient $coinsGeckoClient,
         CoinRepository $coinRepository,
         EntityManagerInterface $em
         )
     {
-        $this->coinsParseService = $coinsParseService;
+        $this->coinsGeckoClient = $coinsGeckoClient;
         $this->coinRepository = $coinRepository;
         $this->em = $em;
     }
 
-    public function create(): void
+    public function create(OutputInterface $output): void
```

Comment:
add $output is problematic when you use this function not in command (e.g. in controller). Maybe add nullable to $output.